functions/billing/test/index.test.js

```diff
@@ -23,7 +23,7 @@ const sinon = require('sinon');
 const cwd = path.join(__dirname, '..');
 
 const PROJECT_NAME = process.env.GOOGLE_CLOUD_PROJECT;
-
+const FF_CMD = 'functions-framework';
```

Comment:
In general we should probably use the absolute path to the binary, not the alias. Might cause fewer issues.

Reply:
See b/170169197 🙂